RoboJackets/robocup-software
# Create Agent Communication System

## Description
This PR creates the basics for agent-to-agent communication.

## Associated / Resolved Issue
[ClickUp card](https://app.clickup.com/t/346bmtz)

## Design Documents
[Link](link-to-design-doc)

## Steps to Test
### Test Case 1
1. Make perf
2. Run framework
3. Make run-sim
4. Observe "SENDING REQUEST TO AGENT: {1...10}" in green (this is a broadcast position request fro
5. 

**Expected result:**???

## Key Files to Review
Group 1
 * File 1
 * File 2

Group 2
 * File 3
 * File 4

## Review Checklist


## (Optional) Sub-issues (for drafts)
_Note: if you find yourself breaking this PR into many smaller features, it may make sense to break up the PR into logical units based on these features._
